jmix-flowui/flowui/src/main/resources/io/jmix/flowui/view/view.xsd

```diff
@@ -48,6 +48,7 @@
             <xs:attribute name="messagesGroup" type="xs:string"/>
             <xs:attribute name="title" type="layout:resourceString"/>
             <xs:attribute name="focusComponent" type="xs:string"/>
+            <xs:attribute name="extends" type="xs:string"/>
```

Comment:
In addition to the `extends` attribute, Jmix classic UI contains additional `xsd` file, called `window-ext.xsd` which contains the `index` attribute. This attribute is used by `XmlInheritanceProcessor` to insert a component to a certain position.

Create a similar file `view-ext.xsd`.

Reply:
done